homework/thinhbka/lesson5-graph/1730. Shortest Path to Get Food.py

```diff
@@ -0,0 +1,26 @@
+DIR = [(0,-1),(0,1),(1,0),(-1,0)]
+class Solution:
+    def getFood(self, grid: List[List[str]]) -> int:
+        rows = len(grid)
+        cols = len(grid[0])
+        visited = [[False for _ in range(cols)] for _ in range(rows)]
+        step = [[0 for _ in range(cols)] for _ in range(rows)]
+        for i in range(rows):
+            for j in range(cols):
+                if grid[i][j] == "*":
```

Comment:
there is only `1` "*"

Reply:
added  **has_found_star = False** to prevent loop after found "*"